options/locale/locale_en-US.ini

```diff
@@ -768,6 +768,17 @@ settings.add_key_success = New deploy key '%s' has been added successfully!
 settings.deploy_key_deletion = Delete Deploy Key
 settings.deploy_key_deletion_desc = Deleting this deploy key will remove all related accesses for this repository. Do you want to continue?
 settings.deploy_key_deletion_success = Deploy key has been deleted successfully!
+protected_branch=Branch protection
+settings.protected_branch=Branch protection
+settings.protected_branch_can_push=Allow push?
+settings.protected_branch_can_push_yes=You can push
+settings.protected_branch_can_push_no=You can not push
+settings.add_protected_branch=Enable protection
+settings.delete_protected_branch=Disable protection
+settings.add_protected_branch_success=Locked successfully
+settings.remove_protected_branch_success=Unlocked successfully
+settings.protected_branch_deletion=To delete a protected branch
+settings.protected_branch_deletion_desc=The branch will be writable. Are you sure?
```

Comment:
`Anyone with write permissions will be able to push directly to this branch. Are you sure?`